src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,33 +1,78 @@
 package edu.kis.vh.nursery;
 
+/**
+ * Basic rhymer type implementation
+ */
 public class DefaultCountingOutRhymer {
 
-    private int[] NUMBERS = new int[12];
-    public int total = -1;
+    private static final int STACK_CAPACITY = 12;
+    private static final int ERROR_CODE = -1;
+    private static final int MAX_SIZE = 11;

```

Comment:
This constant is related to MAX_SIZE. Changing only one leads to an error (inconsistency).